CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
+## 2.0.4
+ - Elasticsearch 2.0 does not allow for dots in field names.  This change changes to use sub-field syntax instead of
```

Comment:
Well, to be fair, I thought of this too.  But since 2.0 isn't released yet, I thought it fair to keep it in a 2.0.x version.

If consensus is to bump to 2.1.0, I will be happy to do so.